hugo/layouts/papers/single.html

```diff
@@ -179,14 +179,14 @@ <h5 class="card-title">Abstract</h5>
         </a>
         {{ end }}
       {{ else }}
-        {{ with $paper.url }}
-        <a class="btn btn-primary" href="{{ . }}"  title="Open PDF of '{{ $paper.title | htmlEscape }}'">
+        {{ with $paper.pdf }}
+        <a class="btn btn-primary" href="{{ . }}" title="Open PDF of '{{ $paper.title | htmlEscape }}'">
```

Comment:
The same change should be made in line 134 above (the "URL:" field on the page).

Reply:
Done.